src/panels/config/lovelace/dashboards/ha-config-lovelace-dashboards.ts

```diff
@@ -59,7 +68,7 @@ export class HaConfigLovelaceDashboards extends LitElement {
             "ui.panel.config.lovelace.dashboards.picker.headers.icon"
           ),
           type: "icon",
-          template: (icon, dashboard) =>
+          template: (icon: DataTableItem["icon"], dashboard: DataTableItem) =>
```

Comment:
I tried to infer the first parameter by improving the `DataTableColumnContainer` type but I did not succeed because we can have custom key and this parameter can be `undefined`. I wonder if we can just drop the first one and keep the second one only (`DataTableItem`). I will be less confusing to use.

Reply:
Yeah, agree